MapboxNavigation/UIView.swift

```diff
@@ -89,53 +89,6 @@ extension UIView {
         return view
     }
     
-    var safeArea: UIEdgeInsets {
-        guard #available(iOS 11.0, *) else { return .zero }
-        return safeAreaInsets
-    }
```

Comment:
min deployment target is iOS12, so no need for this code

Reply:
Yes makes perfect sense. If variables are removed or if parameters of public methods are changed, it would be great if this would be mentioned in the changelog :)

Reply:
> Any type members added in an extension have the same default access level as type members declared in the original type being extended. If you extend a public or internal type, any new type members you add have a default access level of internal. 

The extension doesn't have an access modifier on it, and none of these removed computed props had them either, so I think this should be safe to remove (all were internal and thus not part of the public interface).